static/js/drafts.js

```diff
@@ -498,30 +499,31 @@ function drafts_scroll(next_focus_draft_row) {
     }
     activate_element(next_focus_draft_row[0].children[0]);
 
+    const scroll_element = ui.get_scroll_element($(".drafts-list"));
+
     // If focused draft is first draft, scroll to the top.
     if ($(".draft-info-box").first()[0].parentElement === next_focus_draft_row[0]) {
-        $(".drafts-list")[0].scrollTop = 0;
+        scroll_element.scrollTop(0);
     }
 
     // If focused draft is the last draft, scroll to the bottom.
     if ($(".draft-info-box").last()[0].parentElement === next_focus_draft_row[0]) {
-        $(".drafts-list")[0].scrollTop =
-            $(".drafts-list")[0].scrollHeight - $(".drafts-list").height();
+        scroll_element.scrollTop($(scroll_element).prop("scrollHeight"));
     }
 
     // If focused draft is cut off from the top, scroll up halfway in draft modal.
-    if (next_focus_draft_row.position().top < 55) {
+    if (next_focus_draft_row.position().top < 0) {
         // 55 is the minimum distance from the top that will require extra scrolling.
```

Comment:
Why is this changed from 55 to 0? Also, the comment still refers to 55.

(Same with the −4 to 0 below.)

Reply:
Oh yeah, i forgot to update the comments
I changed this because with 55 it happened, that the focused draft was the second one that was visible (so the top one was fully visible and the next one, the focused draft, was also fully visible) but the scrolling was still triggered.
Same with -4 to 0

Reply:
Can you explain this more clearly?  I don't understand the explanation at all.

Reply:
@timabbott 
So if the user scrolls up or down the drafts using the up/down keys, this is the behavior i would expect:
![chrome-capture (6)](https://user-images.githubusercontent.com/74348920/157852152-a0cb70b9-f74a-4162-9767-4ab6a78510b3.gif)
In this example i am scrolling up (using the up key) and the moment the next focused draft (in this case `bnm`) is not fully visible anymore it is scrolled up so that the focused draft is fully visible and in the middle of the modal (it is in the middle, i just couldn't capture the whole window).
-> This is the case if i use `< 0` in the condition

But with using `< 55` in the condition it sometimes happens that this is the behavior: 
![chrome-capture (7)](https://user-images.githubusercontent.com/74348920/157853025-6d31cb5f-d527-45c5-b5a6-d8544af838cf.gif)
Again i am scrolling up using the up key. 
And i would expect that it is scrolled up once i focus the `ghj` draft, since this is the first draft that is not fully visible anymore.
But as you can see this already happens when focusing the top `fgh` draft, even though this draft is completely visible.

(the same thing applies while scrolling down)

I hope this clears it up a bit!

Reply:
Changes in a pull request should be [grouped logically](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html). If this change from 55 to 0 is a logical part of switching to SimpleBar, you should explain why it’s required for this switch. If this is a separate unrelated change, then you should open it as a separate PR that _doesn’t do anything else_, and explain there why you believe it’s desirable.

Reply:
@andersk i tested it out, and this is indeed only needed after switching to SimpleBar, so i can't really separate this out into another PR.
I don't really understand what you mean with "explain why it's required for this switch", since i already explained that those changes are needed so the behavior explained above doesn't occur.